code/modules/power/supermatter/supermatter.dm

```diff
@@ -251,6 +257,12 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	///Can the crystal trigger the station wide anomaly spawn?
 	var/anomaly_event = TRUE
 
+	var/admin_cascade = FALSE
```

Comment:
whats the point of this? it's never actually set anywhere besides here

Reply:
if an admin wants to VV the SM to ensure a cascade they can, need to doc some of those vars